astropy/units/cds.py

```diff
@@ -36,25 +36,24 @@
 
 __all__ = ["enable"]  #  Units are added at the end
 
-from .core import UnitBase
+import numpy as np
+
+from astropy.constants import si as _si
```

Comment:
It would be possible to write
```suggestion
from astropy.constants import si
```
but then we'd have to go and replace `_si` with `si` in every line it is used. It's not obvious to me that the code churn would be worth it.

Reply:
Agreed to keep it - it is also somewhat handy that the `_si` refers to `constants.si` and not to `units.si`.